src/interfaces.ts

```diff
@@ -330,18 +330,18 @@ export interface IConversationDetails {
     description?: string;
     roles: IConversationRoles;
     isPublic: boolean;
-    participants?: IConversationParticipant[];
 }
 
 /**
  * Conversation details interface
  */
 export interface IConversationDetails2 extends IConversationDetails {
```

Comment:
Why do we have IConversationDetails2 already - can we not safely add these properties to IConversationDetails?